crud/src/main/java/com/redhat/lightblue/assoc/scorers/CostAndSize.java

```diff
@@ -0,0 +1,37 @@
+/*
+ Copyright 2013 Red Hat, Inc. and/or its affiliates.
+
+ This file is part of lightblue.
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.redhat.lightblue.assoc.scorers;
+
+import java.math.BigInteger;
+
+public class CostAndSize {
+    BigInteger cost=BigInteger.ONE;
+    BigInteger size=BigInteger.ONE;
```

Comment:
They are visible, both for read-write. It is an internal class, I can make the class package scope.